libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -255,6 +255,8 @@ namespace eosio { namespace chain {
                                     3030012, "Invalid block extension" )
       FC_DECLARE_DERIVED_EXCEPTION( ill_formed_additional_block_signatures_extension, block_validate_exception,
                                     3030013, "Block includes an ill-formed additional block signature extension" )
+      FC_DECLARE_DERIVED_EXCEPTION( ill_formed_quorum_certificate_extension, block_validate_exception,
+                                    3030013, "Block includes an ill-formed formed quorum certificate extension" )
```

Comment:
I think we can remove this. If not it should have a value of `3030014`.

Reply:
Thanks. This is no longer needed, as we do not check in abi_serilizer any more. Will remove.

Reply:
Removed.